tests/Value/ValueTest.php

```diff
@@ -39,7 +39,7 @@ public function parsesArithmeticInFunctions($operator)
     {
         $subject = Value::parseValue(new ParserState('max(300px, 50vh ' . $operator . ' 10px);', Settings::create()));
 
-        self::assertSame('max(300px,50vh ' . $operator . ' 10px)', (string) $subject);
+        self::assertSame('max(300px,50vh ' . $operator . ' 10px)', (string)$subject);
```

Comment:
This surely contradicts PHP-Fixer, which added spaces after casts.

Reply:
Done.